akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -377,7 +377,8 @@ the `jackson-json` binding the default configuration is:
 
 @@snip [reference.conf](/akka-serialization-jackson/src/main/resources/reference.conf) { #compression }
 
-Messages larger than the `compress-larger-than` property are compressed with GZIP.
+Supported compression algorithms are: gzip, lz4. Use 'off' to disable compression.
+Messages larger than the `compress-larger-than` property are compressed.
```

Comment:
Something about why both of them are options, gzip is less cpu intense but gives worse compression or what is the difference?

Reply:
Well. I just took up the torch left by https://github.com/akka/akka/pull/27131. It is unclear to me as for the motivation of adding compression algorithms. I think the benefit of gzip is that it is built in. The advantage of lz4 is that lz4 is faster.

Reply:
I think that is good enough, please add that.